stdlib/public/runtime/ImageInspectionELF.cpp

```diff
@@ -78,6 +79,7 @@ static int iteratePHDRCallback(struct dl_phdr_info *info,
   return 0;
 }
 
+SWIFT_RUNTIME_EXPORT
```

Comment:
This is used because the functions are in libswiftImageInspectionShared.a but called from libswiftCore.a

Its not needed for the ImageInspectionStatic.cpp functions as this file is not compiled with
 -fvisibility=hidden